Packs/CommonPlaybooks/Playbooks/playbook-Containment_Plan_-_Quarantine_File.yml

```
@@ -0,0 +1,427 @@
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "13":
+    id: "13"
+    taskid: 1208d3af-a6d7-4126-86f2-1c84f8659895
+    type: condition
+    task:
+      id: 1208d3af-a6d7-4126-86f2-1c84f8659895
+      version: -1
+      name: Should quarantine file?
+      description: Whether to quarantine the files based on the input values and the incident context.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "2"
+      "yes":
+      - "8"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: inputs.FileContainment
+            iscontext: true
+          right:
+            value:
+              simple: "True"
+          ignorecase: true
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: inputs.FileRemediation
+            iscontext: true
+          right:
+            value:
+              simple: Quarantine
+          ignorecase: true
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: Core.quarantineFiles.status
+                accessor: status
+            iscontext: true
+          right:
+            value:
+              simple: "False"
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 380
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "15":
+    id: "15"
+    taskid: 410c04ef-0afc-4578-8b1b-b1a3de96debf
+    type: condition
+    task:
+      id: 410c04ef-0afc-4578-8b1b-b1a3de96debf
+      version: -1
+      name: Was the file quarantined?
```

Let's throw an error if the status is 'Failed'.